### PR TITLE
Announce Durham for 2023

### LINKED ISSUE #2: Remove definite 2022 references i.e. San Diego, etc. and maybe create issues to fill in and update those areas as we build out the conference.

### PR DESCRIPTION
**Pull request by drewbrew**
1. Adjust dates (2022->2023, October 16-21 -> October 15-20)
2. Adjust city (San Diego -> Durham)
3. Adjust time zone (Pacific -> Eastern)
4. Add some basic details to the code of conduct page (nearest hospital, police phone numbers, etc)
5. Update the landing page: list the venue name and add a link the call for volunteers blog post
image
image

Fixes #2.

### CODE REVIEW

**Comment by tim-schilling** [comment]
Is it intentional that the other hand has three fingers?

File: _pages/faqs.html [hunk 1]
```diff
@@ -105,9 +105,9 @@ <h2 class="text-center">Opportunity Grants</h2>
     <div class="row column faq-list">
       <section class="faq-item">
         <h2 class="text-center">Sponsors</h2>
-        <p><em>More information is on our <a href="{% link _pages/sponsors-information.md %}">Become a Sponsor</a> page. Sponsorship signups for 2022 are now closed.</em></p>
+        <p><em>More information is on our <a href="{% link _pages/sponsors-information.md %}">Become a Sponsor</a> page. Sponsorship signups for 2023 are now closed.</em></p>
```
Comment (tim-schilling) [inline _pages/faqs.html:108]: Should we remove the mention that sponsorship signups are closed?

**Comment by drewbrew** [comment]
I never noticed that

**Comment by drewbrew** [comment]
It's been the case since the hands were first made in 2018: https://github.com/djangocon/2022.djangocon.us/commit/dc2ef9f009b11f59d5a742bfa64f8bc9238c575b#diff-a301bebfdbc5b31af403779eb00f96f534703c298a7b40586c80d3c65872808d

File: _pages/tickets.html [hunk 2]
```diff
@@ -143,19 +143,19 @@ <h2>Ticket Prices and Types</h2>
    <h2>What is the refund policy?</h2>
    <p>We would hate for you to miss the conference! But we know things come up, so here's our refund policy:</p>
    <ul>
-      <li>Refunds requested before October 1, 2022, incur a 10% cancellation fee.</li>
-      <li>Between October 2 and October 8, 2022, the cancellation fee is 25%.</li>
-      <li>Between October 9 and October 15, the cancellation fee is 50%.</li>
+      <li>Refunds requested before October 1, 2023, incur a 10% cancellation fee.</li>
+      <li>Between October 2 and October 7, 2023, the cancellation fee is 25%.</li>
+      <li>Between October 9 and October 14, the cancellation fee is 50%.</li>
```
Comment (tim-schilling) [inline _pages/tickets.html:148]:
```
      <li>Between October 8 and October 14, the cancellation fee is 50%.</li>
```

File: _pages/faqs.html (hunk 1 above)
Comment (drewbrew) [reply _pages/faqs.html]: Fixed, thanks!

File: _pages/tickets.html (hunk 2 above)
Comment (drewbrew) [reply _pages/tickets.html]: Fixed, thanks!

**Comment by drewbrew** [comment]
and here's the (horribly broken, but that will come in future PRs) venue page

image

**Pull request merged**